.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thank you for your contribution to DeepSensor! -->
+<!-- Please fill out the details in the pull request description below as appropriate -->
+<!-- Ensure that you abide by the code of conduct -->
+
+## :pencil: Description
+<!-- Please provide a clear description of the changes that are introduced in this pull request and why -->
+<!-- It also helps to explain any justification or options you're aware of -->
+<!-- If this pull request relates to an issue, please include the issue number here, e.g. #42 -->
+
+
+## :white_check_mark: Checklist before requesting a review
+(See the contributing guide for more details on these steps.)
+- [ ] I have installed developer dependencies with `pip install -r requirements/requirements.dev.txt` and running `pre-commit install` (or alternatively, manually running `ruff format` before commiting)
+
+If changing or adding source code:
+- [ ] tests are included and are passing (run `pytest`).
+- [ ] documentation is included or updated as relevant, including docstrings.
+
+If changing or adding documentation
+- [ ] docs build successfully (`jupyter-book build docs --all`)
```

Comment:
"and the changes look good from a manual inspection of the HTML in `docs/_build/html/`."